### PR TITLE
BAH-2139| Dockerized atomfeed-console

### PR DESCRIPTION
**Pull request by stavtnameh**
-->added dockerfile and resources
-->added env variables
-->added atomfeed-console to docker-compose
-->added proxy.conf for atomfeed-console
-->added workflow for atomfeed-console docker image

### CODE REVIEW

**Review by mohan-13 — COMMENTED** [review]
The readme needs to be updated in the appropriate places.

File: .github/workflows/atomfeed_console_build_publish.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        uses: actions/setup-java@v2
+        with:
+          distribution: "zulu"
+          java-version: "8"
+      # - name: Run Unit Tests
+      #   run: mvn --no-transfer-progress clean test
+      - name: Install bower
+        run: npm install -g bower 
+      - name: Package
+        run: mvn --no-transfer-progress clean package -DskipTests
```
Comment (mohan-13) [inline .github/workflows/atomfeed_console_build_publish.yaml:29]: Why do we want to build the atomfeed console source code here ? Can we move this step to the atomfeed-console repository itself and consume only the bundled jar file here ?

**Comment by gsluthra** [comment]
@angshu - Please provide your review comments to this PR. The idea is to dockerize atomfeed-console, but since the docker work is Bahmni specific, it was felt that this PR will be raised into Bahmni org and not ICT4H. The assumption is that ICT4H is atomfeed protocol stuff independent of Bahmni domain.. whereas the atomfeed-console DOCKER work is assuming Bahmni in all its connections to OpenMRS/Odoo/etc. 

The docker work here should treat atomfeed-console as a 3rd party product that is being used (just like we would use MySQL or ApacheWeb Server), and package it with appropriate configuration settings for Bahmni SaaS world (or on-prem world).

File: .github/workflows/atomfeed_console_build_publish.yaml (hunk 1 above)
Comment (gsluthra) [reply .github/workflows/atomfeed_console_build_publish.yaml:29]: I also feel the same way. Should treat atomfeed-console as a 3rd party product and consume its jar/images directly as much as possible.

Comment (angshu) [reply .github/workflows/atomfeed_console_build_publish.yaml:29]: yes. absolutely. atomfeed-console should be build as part of its own pipeline/GHA. and here we ought to just use the artefact and create/publish the docker image

File: atomfeed-console/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# atomfeed-console
+
+This directory contains resources for building Docker image for atomfeed-console. The atomfeed-console container is used to show the failed events that happened in an atomfeed client.
+
+Atomfeed console is  is useful when you are facing some sync issues with components like OpenMRS, OpenELIS, Odoo, etc. Issues can be like patients not getting created, orders not getting synced, pacs orders not getting synced etc.,
+
+## Building the image locally
```
Comment (angshu) [inline atomfeed-console/README.md:7]: Should the build instructions not moved to atomfeed-console repo?

Comment (umair-fayaz) [reply atomfeed-console/README.md:7]: @angshu  these build instructions are for building the Docker image and not the atomfeed-console artifact in particular.

File: .github/workflows/atomfeed_console_build_publish.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            echo "setting artifact version to $(echo $APP_VERSION-${{github.run_number}})"
+            echo "ARTIFACT_VERSION=$(echo $APP_VERSION-${{github.run_number}})" >> $GITHUB_ENV
+          fi
+      - name: Download atomfeed-console jar
+        run: |
+          curl -H "Accept: application/vnd.github.v3+json" \
+          https://api.github.com/repos/ICT4H/atomfeed-console/actions/artifacts | \
+          jq -r '.artifacts[0].archive_download_url' | \
+          xargs curl -L -o atomfeed-console-SNAPSHOT.zip -H "Authorization: token ${{secrets.BAHMNI_PAT}}" &&\
+          unzip atomfeed-console-SNAPSHOT.zip  -d atomfeed-console/resoources/
```
Comment (gsluthra) [inline .github/workflows/atomfeed_console_build_publish.yaml:33]: The spelling `resoources` has typo.

**Comment by gsluthra** [comment]
All good from my side. Please review @mohan-13 and @angshu

**Pull request merged**